imports/stylesheets/variables.import.styl

```diff
@@ -79,6 +79,7 @@ $incident-type-colors = {
   'date': lighten($positive, 50%)
   'case': lighten($highlight, 60%)
   'uncertain': darken($l-gray, 5%)
+  'disease': #f1d9eb
```

Comment:
The disease color is actually a little different, it's more purple than red. Since the primary colors are taken, it's hard to make it more distinctive.